### PR TITLE
Update REPORT_TEMPLATE.md

### PR DESCRIPTION
**Pull request by 16647615268**
update Wuhan2020 analysis report

### CODE REVIEW

File: case-study/Wuhan2020/REPORT_TEMPLATE.md [hunk 1]
```diff
@@ -2,6 +2,131 @@
...
+In 20 days, from scratch, from one idea to gather more than 900 contestants and observers, 58 shortlisted teams, more than 100 online interactions, and 8 winning teams.
+
+We conducted a questionnaire survey among the contestants, 80% of the respondents had not participated in hackathons before, 45% were hesitant to participate, 68% believed that industry mentors were the motivation to attract contestants, and 34% were students.
+
+# 中文
+## 介绍
+
+Wuhan2020开源社区于2020年1月24日由同济大学一名博士生发起。
+
+Wuhan2020是COVID-19疫情期间出现的较大规模自组织的代表，也是典型的技术型自组织社区之一。
```
Comment (sunshinemingo) [inline case-study/Wuhan2020/REPORT_TEMPLATE.md:72]: Leave a blank space before and after Chinese and English.
For example, "Wuhan2020是COVID-19疫情" should be "Wuhan2020 是 COVID-19 疫情".

Comment (16647615268) [reply case-study/Wuhan2020/REPORT_TEMPLATE.md:72]: Copy that

**Review by 16647615268 — COMMENTED** [review]
update

**Comment by sunshinemingo** [comment]
Thanks for the update.

**Pull request merged**